src/TheNextWeek/sphere.h

```diff
@@ -43,10 +43,17 @@ bool sphere::bounding_box(double time0, double time1, aabb& output_box) const {
 }
 
 
-void get_sphere_uv(const point3& p, double& u, double& v) {
+static void get_sphere_uv(const point3& p, double& u, double& v) {
```

Comment:
If you want to make the source `static`, you'll need to make the inline src in the text `static`.

Reply:
This really should be private, but everything's awkward with our approach of doing everything in a header.

Reply:
Made private static.